We do not have comrak packaged yet.  It is optional test dependency.

--- readme_renderer-45.0/tox.ini.orig
+++ readme_renderer-45.0/tox.ini
@@ -16,7 +16,6 @@
     pytest
 commands =
     pytest -Wall --strict-markers {posargs}
-extras = md
 
 [testenv:mypy]
 basepython = python3
